### DISCUSSION TITLE
Japanese file names do not have voiced or handakuten marks.

### DISCUSSION THREAD

**Discussion by PCL86** [aclap-dev/video-downloadhelper#738] [Help and Q&A]
### Your question

1. After the U.I. of Video DownloadHelper was updated, voiced and half-voiced marks are no longer added to Japanese file names.
Also, the first character of the file name [Mosaic Destruction] automatically changes to Mosaic.
Returning to Legacy U.I. will restore normality.
2. After using the new "Installed companion app: VdhCoApp 2.0.19", when a download error occurs, it is no longer clear which video is causing the error.
I would appreciate it if you could fix the above two points.

### Addon & CoApp version and details

Video DownloadHelper 9.0.1.13 (stable)
Installed companion app: VdhCoApp 2.0.19

**Comment by Musouheki** [comment]
「1.ファイル名」については #671 で書かれています。
"1. File name" is described in #671.
